app/policies/project_policy.rb

```diff
@@ -2,174 +2,209 @@
 
 # Policy for projects authorization
 class ProjectPolicy < NamespacePolicy # rubocop:disable Metrics/ClassLength
+  def effective_access_level(include_group_links = true) # rubocop:disable Style/OptionalBooleanParameter, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity, Metrics/AbcSize
```

Comment:
**nitpick (blocking):** Let's simplify this and remove the `include_group_links` option, I know this was the old behaviour but it was undocumented so instead of changing documentation let's just allow access regardless of where it came from. In the UI, it does not list the caveat that if from a share you can't do x or y.